q2_phylogeny/plugin_setup.py

```diff
@@ -418,6 +422,41 @@
                  "are not tip identifiers in tree.")
 )
 
+T2 = TypeMatch([Rooted, Unrooted])
+
+T3 = (Frequency | RelativeFrequency | PresenceAbsence | Composition)
+
+plugin.methods.register_function(
+    function=q2_phylogeny.filter_tree,
+    inputs={'tree': Phylogeny[T2],
+            'table': FeatureTable[T3],
```

Comment:
Shouldn't the feature-table be listed under parameters? Since it is set to optional? Or is this a type-map thing?

Reply:
I think this is an artifact thing, since the same syntax shows up in the `fiter-seqs` function where the table is also optional.

https://github.com/qiime2/q2-feature-table/blob/50c0a9c18d9934083a5e719cbef52a40dcf9c9ed/q2_feature_table/plugin_setup.py#L429

The default is `None` in the python file, which should make it optional here